src/class-block-converter.php

```diff
@@ -199,8 +199,11 @@ protected function img( DOMElement|DOMNode $element ): ?Block {
 		if ( empty( $image_src ) && ! empty( $element->getAttribute( 'src' ) ) ) {
 			$image_src = $element->getAttribute( 'src' );
 		}
-
-		$image_src = $this->upload_image( $image_src, $alt ?? '' );
+		try {
```

Comment:
This won't work—`upload_image` can _return_ a `WP_Error`, it doesn't `throw`. We'll need to modify the upstream functions to `throw` instead.

Reply:
Updated in helpers.php. This logic should work now.